content/en/account_management/rbac/permissions.md

```diff
@@ -67,28 +67,29 @@ Find below the list of permissions for the Security Monitoring assets:
-
-| Name                           | Description                                | Scopable |
-| ------------------------------ | ------------------------------------------ | -------- |
-| `logs_read_data`               | Read access to log data                   | true     |
-| `logs_modify_indexes`          | Update the definition of log indexes       | false    |
-| `logs_write_exclusion_filters` | Update indexes exclusion filters           | true     |
-| `logs_write_pipelines`         | Update log pipelines                       | false    |
-| `logs_write_processors`        | Update the log processors in a pipeline    | true     |
-| `logs_write_archives`          | Update the external archives configuration | false    |
-| `logs_read_archives`           | See archive configuration details, access content from the archive | true     |
-| `logs_write_historical_views`  | Rehydrate data from Archives               | false    |
-| `logs_public_config_api`       | Access the Logs Public Config API (r/w)    | false    |
-| `logs_generate_metrics`        | Access the Generate Metrics feature        | false    |
+Find below the list of permissions for the log configuration assets and log data, along with the typical category of user to whom assign this permission (refer to our recommendations on how to assign permissions to team members in the [Logs RBAC][17] guide).
+
+| Name                                                           | Description                                | Scopable |  Typical User |
+| -------------------------------------------------------------- | ------------------------------------------ | -------- | ------------- |
+| [`logs_read_data`](#logs_read_data)                            | Read access to log data                    | true     | Read-Only     |
+| [`logs_modify_indexes`](#logs_modify_indexes)                  | Update the definition of log indexes       | false    | Admin         |
+| [`logs_write_facets`](#logs_write_facets)                      | Create, Update and Delete Log Facets       | false    | Standard      |
+| [`logs_write_exclusion_filters`](#logs_write_exclusion_filters)| Update indexes exclusion filters           | true     | Standard      |
+| [`logs_write_pipelines`](#logs_write_pipelines)                | Update log pipelines                       | false    | Admin         |
+| [`logs_write_processors`](#logs_write_processors)              | Update the log processors in a pipeline    | true     | Standard      |
+| [`logs_write_archives`](#logs_write_archives)                  | Update the external archives configuration | false    | Admin         |
+| [`logs_read_archives`](#logs_read_archives)                    | See archive configuration details, access content from the archive | true     | Standard |
+| [`logs_write_historical_views`](#logs_write_historical_views)  | Rehydrate data from Archives               | false    | Standard      |
```

Comment:
the heading this is pointing to is "view" so the anchor is broken. I suspect the heading should be "views" but leave it to you to figure out which